packages/plugins/Attributions/package.json

```diff
@@ -1,7 +1,8 @@
 {
   "name": "@polar/plugin-attributions",
   "version": "1.2.1",
-  "description": "Displays the Attributions of the Map",
+  "description": "Attributions plugin for POLAR that adds copyright information regarding all currently active layers.",
```

Comment:
```suggestion
  "description": "Attributions plugin for POLAR that adds copyright information regarding all currently active layers. Additionally, static information can be added as well.",
```

Reply:
Removed the `as well` since that's fully expressed by `Additionally`.